src/service/extension-location.js

```diff
@@ -21,14 +21,11 @@ import {getMode} from '../mode';
  * Calculate the base url for any scripts.
  * @param {!Location} location The window's location
  * @param {boolean=} isLocalDev
- * @param {boolean=} isTest
  * @return {string}
  */
-function calculateScriptBaseUrl(location, isLocalDev, isTest) {
+function calculateScriptBaseUrl(location, isLocalDev) {
   if (isLocalDev) {
-    if (isTest || isMax(location) || isMin(location)) {
-      return `${location.protocol}//${location.host}/dist`;
-    }
+    return `${location.protocol}//${location.host}/dist`;
```

Comment:
Agree, DCE of localDev code can be a separate project :-). I do see many code can be DCEed if we simplify our isLocalDev logic [here](https://github.com/ampproject/amphtml/blob/master/src/mode.js#L84)